src/PDS4_NH_IngestLDD.xml

```diff
@@ -174,6 +275,36 @@
     </DD_Value_Domain>
   </DD_Attribute>
   
+  <DD_Attribute>
+    <name>binning_mode</name>
+    <version_id>1.0</version_id>
+    <local_identifier>binning_mode</local_identifier>
+    <nillable_flag>false</nillable_flag>
+    <submitter_name>BJH</submitter_name>
+    <definition>
+      The nh:binning_mode attribute indicates whether LORRI was operating
+      in 1x1 or 4x4 binning mode.
+    </definition>
+    <DD_Value_Domain>
+      <enumeration_flag>true</enumeration_flag>
+      <value_data_type>ASCII_Short_String_Collapsed</value_data_type>
+      <DD_Permissible_Value>
+        <value>1x1</value>
+        <value_meaning>
+          The data were obtained in 1x1 binning mode with image dimensions of
+          either 1028x1024 for raw data or 1024x1024 for partially processed.
+        </value_meaning>
+      </DD_Permissible_Value>
+      <DD_Permissible_Value>
+        <value>4x4</value>
+        <value_meaning>
+          The data were obtained in 4x4 binning mode with image dimensions of
+          either 257x256 for raw data or 256x256 for partially processed.
+        </value_meaning>
+      </DD_Permissible_Value>
+    </DD_Value_Domain>
+  </DD_Attribute>
```

Comment:
Duplicate definition of binning mode attribute above

Reply:
Hm. I wonder why LDDTool didn't complain about these things?

Reply:
Ah - because this happened when I was manually resolving merge conflicts while very tired...

Reply:
done